### DISCUSSION TITLE
目前是否有途径禁用访客登录

### DISCUSSION THREAD

**Discussion by zhiiwg** [usememos/memos#59] [Q&A] [answered]
目前创建用户默认id 102，101 是 guest，是否有办法删除访客并禁用访客登录，或者容器启动时有没有相关配置参数🧐

**Comment by boojack** [comment, marked as answer]
@alim0x FYI https://github.com/justmemos/memos/pull/61

**Reply by boojack** [reply]
Now you can use the `neosmemo/memos:dev` tag to preview, and will be released in next version.

**Reply by beyondtop** [reply]
那如果想保留游客浏览呢？我用的是DEV版的，会不会是这个原因，API调用没有权限？

**Reply by boojack** [reply]
是保留测试用的用户吗？如果你自己部署了，并且启动模式是 prod，那么就可以在设置里，自行添加用户。否则就会自带有测试数据

**Comment by yyysuo** [comment]
最新的dev版本默认已经没有guest用户了。
